sdk/core/azure-core/src/http/log_policy.cpp

```diff
@@ -73,7 +74,8 @@ inline std::string GetRequestLogMessage(
         }
       }
 
-      log << _detail::FormatEncodedUrlQueryParameters(encodedAllowedRequestQueryParams);
+      log << Azure::Core::_detail::FormatEncodedUrlQueryParameters(
```

Comment:
with the using statement above, this isn't needed
```suggestion
      log << detail::FormatEncodedUrlQueryParameters(
```

Reply:
Since we have ::_details in azure core and also in Azure Core Http, and we are `using` both namespaces here, this call becomes ambiguous.

Reply:
That makes sense. Thanks!